packages/ferreiro-server/package.json

```diff
@@ -96,7 +97,6 @@
     "validator": "^9.0.0"
   },
   "devDependencies": {
-    "concurrently": "^4.1.0",
```

Comment:
I think this should still be a devDep, since it is not used within the actual app, but rather in the build process.